packs/maven-java11/.lighthouse/jenkins-x/pullrequest.yaml

```diff
@@ -18,7 +18,7 @@ spec:
               -XX:AdaptiveSizePolicyWeight=90 -Xms10m -Xmx192m
           - name: MAVEN_OPTS
             value: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3
-          image: uses:jenkins-x/jx3-pipeline-catalog/tasks/maven-java11/pullrequest.yaml@versionStream
+          image: uses:Mentor-Medier/jx3-pipeline-catalog/tasks/maven-java11/pullrequest.yaml@versionStream
```

Comment:
This always needs to be jenkins-x.

Reply:
This Pr should not be here at all. sorry. Big fingers